suricata/update/main.py

```diff
@@ -692,7 +692,14 @@ def build_rule_map(rules):
         if rule.id not in rulemap:
             rulemap[rule.id] = rule
         else:
+            if rule["rev"] == rulemap[rule.id]["rev"]:
+                logger.warning(
+                    "Found duplicate rule ID {}, keeping existing rule."
+                    .format(rule.id))
```

Comment:
Please use rule SID in the error message. Read about SIDs here: https://suricata.readthedocs.io/en/suricata-4.1.3/rules/meta.html#sid-signature-id